common/v2/utils/index.ts

```diff
@@ -25,3 +25,4 @@ export {
 export { makeBlob } from './blob';
 export { default as consoleAdvertisement } from './consoleAdvertisement';
 export { fromTxReceiptObj } from './transaction';
+export * from './uuidUtils';
```

Comment:
Please favor explicit exports for the module.
It facilitates refactors by having an explicit list of the methods that are used.